examples/gatsby/package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "dependencies": {
     "@emotion/react": "latest",
+    "@emotion/server": "latest",
     "@emotion/styled": "latest",
     "@material-ui/core": "next",
     "gatsby": "latest",
```

Comment:
Ah, wait. I got this error when I'm trying to run `yarn develop` or `yarn build`. add this fix the problem.

```json
"browserslist": [
    "defaults",
    "not IE 11",
    "maintained node versions"
  ]
```

not sure if it needs to be in the example or not?

Reply:
Hmm, I haven't encountered this error. It's probably because you tried to run the example directly in the project. We don't support this. I am usually copying the example folder to a different location and I am running there. Could you confirm that this was the reason?

Reply:
Yes, I ran inside the project. Got it, let's ignore the error I got.